src/api/helpers/decrypt-financial-uk/index.ts

```diff
@@ -0,0 +1,36 @@
+import { ApplicationFinancialUk } from '../../types';
+import decryptData from '../decrypt';
+
+/**
+ * decryptFinancialUk
+ * decrypts the accountNumber and sortCode in the financialUk section of application
+ * returns decrypted data
+ * @param {ApplicationFinancialUk} applicationFinancialUk: financialUk data from application
+ * @returns {ApplicationFinancialUk} decrypted accountNumber and sortCode in financialUk section
+ */
+const decryptFinancialUk = (applicationFinancialUk: ApplicationFinancialUk) => {
```

Comment:
Should this be wrapped inside `try-catch` block?

Reply:
Maybe we should add a `console.info` just after the `try`